src/pykeen/typing.py

```diff
@@ -94,3 +95,4 @@ class ScorePack(NamedTuple):
 
 
 Sign = Literal[-1, 1]
+Mode = Optional[Literal["train", "valid", "test"]]
```

Comment:
[the factories](https://github.com/pykeen/pykeen/blob/16909b03ca810e18669a172a9b0309127fa69e3d/src/pykeen/datasets/base.py#L72-L77) are called "train**ing**", "valid**ation**" and "test**ing**", so maybe we should stick to that here too

Reply:
Renamed them in https://github.com/pykeen/pykeen/pull/722/commits/2cf7bdff63694924c9700bd3c948455ae70def58